.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 8.4.0
+  - 8.9.3
```

Comment:
Let's update the `.nvmrc` file as well so people using nvm will have the correct version.